src/org/exist/storage/DBBroker.java

```diff
@@ -116,6 +116,8 @@
 
     protected IndexController indexController;
 
+    long config_ts = 0;
```

Comment:
I'd like to discuss, but IMO these values need to be read/set with getters and setters.....

Reply:
I would also like to see getters where the class is not an inner class; Otherwise I feel that it breaks encapsulation.

Reply:
why do it needs getter or setter if that `protected` and for in-package use only?

Reply:
I guess there a quite reasons I could think of. Good API design is one of it. There is not really a good reason not to use getters/setters IMO.  These protected members are not accessed directly I assume?